addons/fastroping/functions/fnc_deployRopes.sqf

```diff
@@ -25,6 +25,57 @@ private _hookAttachment = _vehicle getVariable [QGVAR(FRIES), _vehicle];
 {
     private _ropeOrigin = _x;
     private _hook = QGVAR(helper) createVehicle [0, 0, 0];
+    switch (_ropelength) do {
+        case 12: {
+            _ropelength = 12.2;
+            if ("ACE_rope12" in (backpackItems ACE_player)) then {
+                ACE_player removeItemFromBackpack "ACE_rope12";
+            };
+            if ("ACE_rope12" in (ItemCargo _vehicle)) then {
+                _vehicle removeItem "ACE_rope12";
+            };
+        };
+        case 15:{
+            _ropelength = 15.2;
+            if ("ACE_rope15" in (backpackItems ACE_player)) then {
+                ACE_player removeItemFromBackpack "ACE_rope15";
+            };
+            if ("ACE_rope15" in (ItemCargo _vehicle)) then {
```

Comment:
You are removing the rope twice.
If the player uses the rope from his backpack. Why remove the one from backpack AND the one in vehicle cargo. You are only deploying one rope, not two.